support-modules/bigquery/src/main/scala/com/gu/support/acquisitions/BigQueryService.scala

```diff
@@ -0,0 +1,47 @@
+package com.gu.support.acquisitions
+
+import com.google.auth.oauth2.ServiceAccountCredentials
+import com.google.cloud.bigquery.{BigQueryException, BigQueryOptions, InsertAllRequest, TableId}
+import com.gu.monitoring.SafeLogger
+import com.gu.monitoring.SafeLogger.Sanitizer
+import com.gu.support.config.BigQueryConfig
+
+import scala.collection.JavaConverters._
+
+class BigQueryService(config: BigQueryConfig) {
+  lazy val bigQuery =
+    BigQueryOptions
+      .newBuilder()
+      .setProjectId(config.projectId)
+      .setCredentials(ServiceAccountCredentials.fromPkcs8(
+        config.clientId,
+        config.clientEmail,
+        config.privateKey,
+        config.privateKeyId,
+        Nil.asJavaCollection
+      ))
+      .build().getService
+
+  def tableInsertRow(datasetName: String, tableName: String, acquisitionDataRow: AcquisitionDataRow): Either[String, Unit] = {
+    try {
+      val tableId = TableId.of(datasetName, tableName)
+      val rowContent = AcquisitionDataRowMapper.mapToTableRow(acquisitionDataRow)
+      val insertRequest = InsertAllRequest.newBuilder(tableId).addRow(rowContent).build
+
+      val response = bigQuery.insertAll(insertRequest) // Seems there is currently no Async way to do this
```

Comment:
if you want to get control back straight away so you can run something else concurrently you can do something like this
```suggestion
      Future(bigQuery.insertAll(insertRequest)).map { response =>
```